### PR TITLE
Create mutations for object

### PR DESCRIPTION
**Pull request by letrungkien0210**
Hi @jakubknejzlik,

Please review.

### CODE REVIEW

File: lib/graphql.schema.js [hunk 1]
```diff
@@ -137,6 +162,76 @@ const _getRelationshipFromDescription = relationship => {
...
+      const promiseCreateObject = ctx.context.create(entity.name, args.input);
+      ctx.context.save();
+      return promiseCreateObject;
+    }
+  }
+}
+
+const _updateEntity = entity => {
+  const entityInputUpdateType = new GraphQLInputObjectType({
+    name: `${entity.name}InputUpdateType`,
```
Comment (jakubknejzlik) [inline lib/graphql.schema.js]: Please use `${entity.name}UpdateInputType`

File: lib/graphql.schema.js [hunk 2]
```diff
@@ -137,6 +162,76 @@ const _getRelationshipFromDescription = relationship => {
   }
 };
 
+const _createEntity = entity => {
+  const entityInputType = new GraphQLInputObjectType({
+    name: `${entity.name}InputType`,
+    fields: () => (_getSchemaFieldsFromEntityWithoutRelationship(entity))
```
Comment (jakubknejzlik) [inline lib/graphql.schema.js]: `_getSchemaFieldsFromEntityWithoutRelationship` returns also id which should not be possible to specify when creating entity

File: lib/graphql.schema.js [hunk 3]
```diff
@@ -137,6 +162,76 @@ const _getRelationshipFromDescription = relationship => {
   }
 };
 
+const _createEntity = entity => {
+  const entityInputType = new GraphQLInputObjectType({
+    name: `${entity.name}InputType`,
```
Comment (jakubknejzlik) [inline lib/graphql.schema.js]: Please use `${entity.name}CreateInputType`

**Comment by letrungkien0210** [comment]
Hi @jakubknejzlik ,

I fixed your comments.
Please review.

Thanks.

File: test/index.js [hunk 4]
```diff
@@ -3,28 +3,30 @@ const fs = require("fs");
...
 const lib = require("../index");
 
 const database = new CoreData("sqlite://:memory:");
 database.createModelFromYaml(fs.readFileSync(__dirname + "/schema.yml"));
 
 const app = express();
+app.use(bodyParser.json());
 app.use("/graphql", lib.graphql(database));
 
 const test = supertest(app);
```
Comment (jakubknejzlik) [inline test/index.js]: Please remove the log

File: test/index.js [hunk 5]
```diff
@@ -60,9 +62,203 @@ describe("graphql", () => {
...
+        }
+      }
+    };
+
+    return test
+      .post(`/graphql?`)
+      .send(postData)
+      .expect(200)
+      .then(res => {
+        console.log(JSON.stringify(res.body));
```
Comment (jakubknejzlik) [inline test/index.js]: Please remove the console.log

**Comment by letrungkien0210** [comment]
Hi @jakubknejzlik,

I removed the console.log
Please merge.

Thanks.

**Pull request merged**